beetmoverscript/task.py

```diff
@@ -43,6 +43,13 @@ def validate_task_scopes(context, manifest):
                 sys.exit(3)
 
 
+def add_checksums_manifest_to_artifacts(context, filename):
+    abs_file_path = os.path.join(context.config['artifact_dir'],
+                                 'public/{}'.format(filename))
```

Comment:
We can put this in `public/`, or a subdir of `public/`.  I'm not sure which makes more sense.  This works.